<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>100</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавить цель</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/SmartMoney.png</normaloff>../icons/SmartMoney.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
    font: 10pt &quot;MS Shell Dlg 2&quot;;
    color: rgb(255, 255, 255);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0.472, x2:1, y2:0.966, stop:0 rgba(0, 62, 88, 0), stop:1 rgba(133, 64, 158, 0));
}
QDoubleSpinBox {
    font: 10pt &quot;MS Shell Dlg 2&quot;;
    border: 1px solid white;
	color: rgb(255, 255, 255);
	background-color: rgb(70, 70, 70);
}
QDialog {
    background-color: qlineargradient(spread:pad, x1:0, y1:0.108, x2:1, y2:0.489, stop:0 rgba(97, 66, 125, 255), stop:0.576271 rgba(45, 45, 45, 255));
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="suffix">
      <string> ₽</string>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>999999999.990000009536743</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Новая цель:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
